tests/build.rs

```diff
@@ -0,0 +1 @@
+fn main() {}
```

Comment:
Why is this needed?

Reply:
Needed in an older version, but I forgot to delete it.